src/context/ThemeContext.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const lightTheme = {
+  isDarkMode: false,
+  color: '#000000',
+  backgroundColor: '#FFFFFF',
+  cardTextColor: '#757575',
+};
+
+const darkTheme = {
+  isDarkMode: true,
+  color: '#FFFFFF',
+  backgroundColor: '#303030',
+  cardBackgroundColor: '#424242',
+  cardTextColor: '#C7C7C7',
+};
+
+const ThemeContext = React.createContext({});
```

Comment:
here you can use a theme from the ones you defined previously, for example:

```jsx
const ThemeContext = React.createContext(lightTheme);
```

Reply:
@PacoMojica, I used that, but for some reason, when I retrieved the context in another component, it was null. Is there a reason this happened?